packages/a/ag-grid-enterprise.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "ag-grid-enterprise",
+  "filename": "ag-grid-enterprise.min.js",
+  "description": "Advanced Framework Agnostic Javascript Datagrid.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ag-grid/ag-grid"
+  },
+  "keywords": [
+    "ag",
+    "ag-grid",
+    "ag-grid-enterprise",
+    "datagrid",
+    "data-grid",
+    "datatable",
+    "data-table",
+    "grid",
+    "table",
+    "react",
+    "angular",
+    "angular-component",
+    "react-component",
+    "angularjs",
+    "reactjs"
+  ],
+  "license": "MIT",
```

Comment:
This does not seem correct?

Reply:
I've updated it say `Commercial see LICENSE.txt`
I based it on the example similar to [datatables.net-editor-bs4.json](https://github.com/cdnjs/packages/blob/master/packages/d/datatables.net-editor-bs4.json#L9).